gunicorn/arbiter.py

```diff
@@ -424,7 +425,7 @@ def murder_workers(self):
         """
         if not self.timeout:
             return
-        workers = list(self.WORKERS.items())
+        workers = list(self.WORKERS.items() + self.DYING_WORKERS.items())
```

Comment:
Unfortunately, this will raise `TypeError` in Python 3. I'd write this like

```py
workers = list(self.WORKERS.items()) + list(self.DYING_WORKERS.items())
```